include/CppUTestExt/MockNamedValue.h

```diff
@@ -154,6 +162,12 @@ class MockNamedValue
         unsigned int unsignedIntValue_;
         long int longIntValue_;
         unsigned long int unsignedLongIntValue_;
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
Why is this one still needed?

Reply:
You're right it's probably not, will fix.

Reply:
Turns out, this is required because otherwise the union is "non trivial", and the constructors don't work properly.

Reply:
Yea it's a bit ugly, but I didn't have a better solution.